people/mark-assad.md

```diff
@@ -0,0 +1,8 @@
+Marc Alassad
+------------------
+
+![](photos/markassad.JPG)
```

Comment:
Please rename the JPG file to mark-assad.jpg to be consistent with the rest of the names.